### DISCUSSION TITLE
the cursor position in android studio logcat is not correct, how to fix

### DISCUSSION THREAD

**Discussion by quekangkang** [JetBrains/ideavim#977] [Q&A]
I found the cursor in android studio logcat is not in the end , though I move the mouse click the end. 
When I changed to insert mode , or disable ideavim, the cursor position is correct in the end.
If someone can tell me how to solve this, I would be extremely grateful.

![image](https://github.com/user-attachments/assets/977213fc-8ea6-4e0c-9efb-83b02f60acf0)

Here is Android Studio information

Android Studio Koala | 2024.1.1 Patch 2
Build #AI-241.18034.62.2411.12169540, built on August 1, 2024
Runtime version: 17.0.11+0--11852314 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10.0
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 8
Registry:
  ide.instant.shutdown=false
  ide.browser.jcef.gpu.disable=true
  debugger.new.tool.window.layout=true
  ide.experimental.ui=true
  terminal.new.ui=true
  ide.images.show.chessboard=true
  runtime.chooser.url=https://s3-kp-kwe-gen.hics.huawei.com/cleancode-prod/ECodeWorkbench/JBR/jbr-choose-runtime-231.json.xz
Non-Bundled Plugins:
  com.huawei.intellij.plugin.foundation (1.12.10)
  IdeaVIM (2.15.3)
  com.huawei.idea.client (3.9.0)

**Comment by citizenmatt** [comment]
I'm not entirely sure what you're asking for here, sorry. Do you want to position the caret after the last character with the mouse, while in Normal mode? This isn't really possible, and is normal Vim behaviour - in Normal mode, there's nothing you can do _after_ the end of a line. You can only really switch to Insert mode, which you say works as expected.

Could you explain your scenario a little more, please?

**Reply by quekangkang** [reply]
I apologize, I didn't explain it clearly.

“Do you want to position the caret after the last character with the mouse, while in Normal mode? ”
-- Yes.  Normal mode is not work in logcat filter in Android studio, I can only operate as insert mode even in Normal mode. 
so I'd  like the caret will be the end of line when I click the end. (now I have to move the caret by Right key)
    Or, make the Normal mode work in logcat filter. (This is also acceptable.）
